torch/_torch_docs.py

```diff
@@ -3860,6 +3860,34 @@ def merge_dicts(*dicts):
     tensor([ 0.8722, -0.7416,  0.2653, -0.1584])
 """.format(**single_dim_common))
 
+add_docstr(torch.maximum, r"""
+maximum(input, other, *, out=None) -> Tensor
+
+Compares the element-wise maximum of :attr:`input` and :attr:`other`.
+
+.. note::
+    If one of the elements being compared is a NaN, then that element is returned.
+    If both elements are NaNs then the first is returned.
```

Comment:
This part of the note may be removed.

Reply:
The `If both elements are NaNs then the first is returned.` is removed and the whole note is changed like `minimum` function.
Please kindly confirm.